includes/theme-support/class-llms-theme-support.php

```diff
@@ -29,6 +29,184 @@ public function __construct() {
 		add_action( 'after_setup_theme', array( $this, 'includes' ) );
 	}
 
+	/**
+	 * Retrieve formatted inline CSS for a give list of selectors and rules
+	 *
+	 * @since [version]
+	 *
+	 * @param string[] $selectors       Array of CSS selectors.
+	 * @param string[] $rules           Associative array of CSS rules and properties. For example: `array( 'color' => '#fff' )`.
+	 * @param string   $selector_prefix A css selector to prefix each item in $selectors with.
+	 * @return string
+	 */
+	public static function get_css( $selectors, $rules, $selector_prefix = '' ) {
+
+		// Convert the $rules array to a list of CSS strings.
+		$rules_list = array();
+		foreach ( $rules as $prop => $val ) {
+			$rules_list[] = sprintf( '%1$s: %2$s;', $prop, $val );
+		}
+
+		// When supplied, prefix each selector.
+		if ( $selector_prefix ) {
+			foreach ( $selectors as &$selector ) {
+				$selector = $selector_prefix . ' ' . $selector;
+			}
```

Comment:
Look at this beauty:
```suggestion
			$selector = preg_filter( '/^/', $selector_prefix, $selector );
```

Reply:
no.

Reply:
ahah <3